<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseValidationDialog</class>
 <widget class="QDialog" name="LicenseValidationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>License validation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="validation_dialog_resources.qrc">
    <normaloff>:/icons/IMG/validation.png</normaloff>:/icons/IMG/validation.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEdit_license_key">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_expiration">
     <property name="text">
      <string>Expiration date</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="validation_dialog_resources.qrc">
         <normaloff>:/icons/IMG/cancel.png</normaloff>:/icons/IMG/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_OK">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="validation_dialog_resources.qrc">
         <normaloff>:/icons/IMG/ok.png</normaloff>:/icons/IMG/ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_key_icon">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_user_icon">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QLineEdit" name="lineEdit_license_hash">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_license_key">
     <property name="text">
      <string>License key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_user_name">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_user_name">
     <property name="text">
      <string>User name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_license_hash">
     <property name="text">
      <string>License hash</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_time_icon">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_hash_icon">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="2">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="validation_dialog_resources.qrc"/>
 </resources>
 <connections/>
</ui>
